packages/tofu/src/util.js

```diff
@@ -122,6 +124,14 @@ function mergeConfig (configA, configB) {
   return newConfig
 }
 
+function mergeConfig (configA, configB) {
+  configA = JSON.parse(configA)
```

Comment:
why are we calling parse here? that wasn't part of the original fn. plz remove

Reply:
because we're passing in the string from readfile. Unless we want to use parse on the consumer side

Reply:
parse on consumer side